### PR TITLE
13 create makefile

### PR DESCRIPTION
**Pull request by UlkaBo**
source - our project diet. It was corrected.

### CODE REVIEW

File: app/base_function/translator.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
-from settings import settings  # noqa !!!used to load the environment variables required for the function get_translateimporta
 from google.cloud import translate_v2 as translate
-
-from scheme.transdata import TranslateResponse, TranslateRequest
+from scheme.transdata import TranslateRequest, TranslateResponse
+from settings import \
+    settings  # noqa !!!used to load the environment variables required for the function get_translateimporta
```
Comment (ykshatroff) [inline app/base_function/translator.py:4]: To avoid this ugly linebreak, place the comment on a separate line before the import.

File: mypy.ini [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ignore_missing_imports = True
+
+[mypy-pandas.*]
+ignore_missing_imports = True
+
+[mypy-tg_react.*]
+ignore_missing_imports = True
+
+[mypy-tg_utils.*]
+ignore_missing_imports = True
```
Comment (ykshatroff) [inline mypy.ini:57]: These lines are unnecessary.

**Review by ykshatroff — CHANGES_REQUESTED** [review]
The directory `.data/cache` should not be here. Please add it to `.gitignore` and remove from this PR. :)

File: tests/test_get_translate.py [hunk 3]
```diff
@@ -1,39 +1,64 @@
...
+
 import pytest
 from pydantic import ValidationError
-from unittest.mock import patch
-from app.base_function.translator import translate_client
 
-from app.base_function.translator import get_translate
+from app.base_function.translator import get_translate, translate_client
 from app.scheme.transdata import ISO639_1, TranslateRequest
+from app.settings import \
```
Comment (ykshatroff) [inline tests/test_get_translate.py:8]: Same as in app/base_function/translator.py, can be applied here.

**Review by ykshatroff — APPROVED** [review]

File: app/Makefile [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+.PHONY:
+isort: args ?= .
+isort: ## Fix imports automatically with isort
+	echo -e "$(CYAN)Fixing imports with isort$(COFF)"
+	make -s run-python cmd="isort $(args) --skip-glob='*/site-packages/*'"
+
+
+.PHONY:
+psql:
+	docker-compose exec postgres psql --user $(PROJECT_NAME) --dbname $(PROJECT_NAME)
```
Comment (YanaAnisimova) [inline app/Makefile:102]:
```suggestion
	docker-compose exec postgres psql --user $(POSTGRES_USER) --dbname $(POSTGRES_DATABASE_NAME)
```
and need to include .env in Makefile.

**Pull request merged**